rpc/core/README.md

```diff
@@ -0,0 +1,15 @@
+# Tendermint RPC
+
+## Generate markdown for [Slate](https://github.com/tendermint/slate)
+
+We are using [Slate](https://github.com/lord/slate) to power our RPC
+documentation. If you are changing a comment, make sure to copy the resulting
+changes to the slate repo and make a PR
+[there](https://github.com/tendermint/slate) as well. For generating markdown
+use:
+
+```shell
+go get github.com/melekes/godoc2md
```

Comment:
why the fork? is the flag change essential / are you planning on PR'ing upstream?